lib/core/pagination/pagination-component.interface.ts

```diff
@@ -15,17 +15,10 @@
  * limitations under the License.
  */
 
-/**
- * PaginationQueryParams object is used to emit events regarding pagination having two
- * properties from the Pagination interface found in AlfrescoJS API
- *
- * The two properties are "skipCount" and "maxItems" that are sent as query parameters
- * to server to paginate results
- *
- * @TODO Contribute this to AlfrescoJS API
- */
+import { PaginatedComponent } from './paginated-component.interface';
+import { Pagination } from 'alfresco-js-api';
 
-export interface PaginationQueryParams {
-    skipCount: number;
-    maxItems: number;
+export interface PaginationComponentInterface {
```

Comment:
I think adding "Interface" to the name of the interface is against all common practices. Also violates the https://angular.io/guide/styleguide#interfaces style guide. Otherwise we have to add "Class" and "Enum" to all classes and enums.

Reply:
Ok